### DISCUSSION TITLE
Candidatura di: Riccardo Genova

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#51] [Candidature Ambassadors In Corso]
Questo sondaggio viene aperto per tutto il mese di Settembre per indicare la preferenza o l'opposizione sulla candidatura a ruolo di Ambassador per:

[Riccardo Genova](https://github.com/riccardogenova-bitrocket)

In caso di disaccordo, si protegge l'anonimato ma sarebbe gradita una spiegazione tramite commento a questo sondaggio con delle motivazioni.

Attualmente, vista la fase embrionale del progetto, non ci saranno limiti minimi di votazione e la semplice superiorità dei voti a favore sarà considerata sufficiente a procedere con la nomina.

**Comment by eppak** [comment]
Ciao @riccardogenova-bitrocket,
Siamo spiacenti di comunicarti che non hai raggiunto i requisiti necessari per esser accettato come Ambassador.
I requisiti minimi che sono stati definiti dalla community sono:
- Voti positivi superiori di quelli negativi.  

Siamo comunque felici di ricevere le tue contribuzioni al Libro.  
Puoi trovare tutti gli issue e discussion aperte sulla repository.
Grazie per la tua candidatura, ti aspettiamo!
